packages/backend/jest.setup.js

```diff
@@ -1,12 +1,16 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const Knex = require('knex')
 // eslint-disable-next-line @typescript-eslint/no-var-requires
+const { cpus } = require('os')
+// eslint-disable-next-line @typescript-eslint/no-var-requires
 const { GenericContainer } = require('testcontainers')
 
 const POSTGRES_PORT = 5432
 const REDIS_PORT = 6379
 
 module.exports = async () => {
+  const workers = cpus().length - 1
```

Comment:
This appears to work (`globalConfig.maxWorkers` is `7` for me)

```suggestion
module.exports = async (globalConfig) => {
  const workers = globalConfig.maxWorkers
```